src/Illuminate/Database/Schema/Grammars/PostgresGrammar.php

```diff
@@ -876,18 +876,27 @@ protected function typeMultiPolygon(Fluent $column)
      */
     protected function typeMultiPolygonZ(Fluent $column)
     {
-        return $this->formatPostGisType('multipolygonz');
+        return $this->formatPostGisType('multipolygonz', $column);
     }
 
     /**
      * Format the column definition for a PostGIS spatial type.
      *
      * @param  string  $type
+     * @param  Fluent  $column
      * @return string
      */
-    private function formatPostGisType(string $type)
+    private function formatPostGisType(string $type, Fluent $column)
     {
-        return "geography($type, 4326)";
+        if (is_null($column->geography)) {
```

Comment:
Maybe it will be better to refactored this conditionals to remove nesting using early return?

Reply:
Also, as I remember there was PR which replaced `is_null` with `=== null` all over the framework.

Reply:
Can definitely refactor, and will.

I added your recommendation of replacing is_null with === null and !is_null with !== null, even though it isn't on the 6.x branch yet. Either way it should be pretty easy to change now.